### PR TITLE
Lg/replace networking

### PR DESCRIPTION
**Pull request by lsgunnlsgunn**
## Description

https://jira.mesosphere.com/browse/DCOS-47876

## Urgency
- [ X ] Blocker 
- [ ] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects).
- Change all affected versions (e.g. 1.7, 1.8, 1.9, 1.10, 1.11, 1.12).
- See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing).

### CODE REVIEW

**Comment by lsgunnlsgunn** [comment]
Here is the draft for the cluster identity section in the opening page Networking page.

File: pages/1.11/networking/index.md [hunk 1]
```diff
@@ -81,6 +82,23 @@ While both Marathon-LB and Edge-LB are designed for handling north-south ingress
 | zero-hop load balancing               |     X     |         |             |
 | No single point of failure         |     X     |         |             |
 
+# <a name="cluster-id"></a>Specifying a cluster identity for network connections
+The DC/OS networking component (`dcos-net`) supports setting a **cluster identity** option on a node for DC/OS cluster. By enabling this feature, you can prevent nodes from communicating across clusters when a node is moved from one cluster to another. This feature ensures that the nodes from a cluster has a unique identifier and prevents unauthorized "cross-talk" between clusters.
+
+To use thw cluster identity feature:
+1. Edit the `config.yml` file for each node in the cluster to add the `dcos_net_cluster_identity` configuration parameter.
+
+1. Set the parameter value to `true` to enable the use of a cluster identity.
```
Comment (razi-malik) [inline pages/1.11/networking/index.md]: Should this bullet be also '1.'? Looks like a typo

File: pages/1.12/networking/index.md [hunk 2]
```diff
@@ -81,6 +82,23 @@ While both Marathon-LB and Edge-LB are designed for handling north-south ingress
 | zero-hop load balancing               |     X     |         |             |
 | No single point of failure         |     X     |         |             |
 
+# <a name="cluster-id"></a>Specifying a cluster identity for network connections
+The DC/OS networking component (`dcos-net`) supports setting a **cluster identity** option on a node for DC/OS cluster. By enabling this feature, you can prevent nodes from communicating across clusters when a node is moved from one cluster to another. This feature ensures that the nodes from a cluster has a unique identifier and prevents unauthorized "cross-talk" between clusters.
+
+To use thw cluster identity feature:
+1. Edit the `config.yml` file for each node in the cluster to add the `dcos_net_cluster_identity` configuration parameter.
+
+1. Set the parameter value to `true` to enable the use of a cluster identity.
```
Comment (razi-malik) [inline pages/1.12/networking/index.md]: Should this bullet be also '1.'? Looks like a typo

File: pages/1.11/networking/index.md (hunk 1 above)
Comment (lsgunnlsgunn) [reply pages/1.11/networking/index.md]: It is a markdown thing. If you view with the Display rich diff option (toggle between [<>] and [pg] icons), it shows steps (1 and 2)

File: pages/1.12/networking/index.md (hunk 2 above)
Comment (lsgunnlsgunn) [reply pages/1.12/networking/index.md]: It is a markdown thing. If you view with the Display rich diff option (toggle between [<>] and [pg] icons), it shows steps (1 and 2)

**Pull request merged**